org.gnome.BreakTimer.json

```diff
@@ -0,0 +1,87 @@
+{
+    "id" : "org.gnome.BreakTimer",
```

Comment:
Is it a GNOME app that can use org.gnome prefix?

Reply:
It should be (historically it has been, and it still lives in https://gitlab.gnome.org/GNOME), but you've reminded me now is the best time to make sure that is in fact the right thing to do. I'll check on that and let you know in a bit, and indeed should avoid publishing this until then. Thanks for the quick review! :)